identity/app/views/profile/emailPrefs.scala.html

```diff
@@ -43,28 +43,34 @@ <h2 class="form__heading">Your newsletters</h2>
 
         <fieldset class="fieldset">
             <div class="fieldset__heading">
-                <h2 class="form__heading">Options</h2>
```

Comment:
Just curious, why this wording change?

Reply:
Just seemed to better suit the what's in this area for now